tests/acceptance/common.py

```diff
@@ -340,7 +343,9 @@ def common_boot_from_internal(conn=None):
 
 
 def latest_build_artifact(builddir, extension):
-    if pytest.config.getoption('--test-conversion'):
+    global configuration
+    
+    if "conversion" in configuration:
```

Comment:
This way of checking a dictionary key is dangerous, as I could very well have a dict like `configuration = {"conversion": False}` and the check will return `True`.

I suggest `if configuration.get("conversion")`.

Reply:
(also two more times below)

Reply:
I can change it, but we are adding the item to the dictionary only when it is set up. We are safe in this case as we are controlling where the items are added.

Reply:
I know we are safe for now, but it can easily become a problem in the future if someone wants to change the value from another fixture or something.

I saw you changed it in most of the places. Thanks